AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.html

```diff
@@ -170,6 +165,12 @@ <h2 class="header-title">{{ header }}</h2>
                             {{ Device.createdByDevice }}
                         </td>
                     </ng-container>
+                    <ng-container matColumnDef="id">
+                        <th mat-header-cell *matHeaderCellDef>ID</th>
+                        <td mat-cell *matCellDef="let Device" data-label="ID:">
+                            {{ Device.id }}
+                        </td>
+                    </ng-container>
```

Comment:
Could you add a screenshot that shows the table with the new column order please?

Reply:
Now that I see the screenshot... I think it doesn't make sense to have the ID column at the end. 🙈 The problem is that then the "Username" is the first column. So if you just look at the table, you might think that it is a list of users.
Sorry, I should have thought of that.

Reply:
The ID column is still not the first one, is it?

Reply:
Maaaan, why do you keep resolving this comment, even though you didn't fix it??

Reply:
The ID column is the first one, the only part of the code that i truly needed to change is this 
<img width="458" alt="image" src="https://github.com/nmshd/backbone/assets/37671301/0399b319-b4a2-4642-bce1-bcafef4f193c">
as long i change around this array the column representation will change too, it doesn't matter where i put that ng-container as long as it is in the same table where it is used. PS. sorry for not updating the picture in the PR description.

Reply:
In that case, at least a comment would be helpful.

But still, I think I would update the html as well, to avoid confusion.

Reply:
I'll update the html as well.